test/hotspot/jtreg/compiler/lib/ir_framework/TestFramework.java

```diff
@@ -323,6 +324,15 @@ public TestFramework addScenarios(Scenario... scenarios) {
         return this;
     }
 
+    /**
+     * Add test classes to boot classpath. This adds all classes found on path {@link jdk.test.lib.Utils#TEST_CLASSES}
+     * to the boot classpath with "-Xbootclasspath/a". This is useful when trying to run tests with @Stable annotations.
```

Comment:
Again, `@Stable` is overly specific here. This is "just" about the privileged code.